### PR TITLE
The Messenger: docs improvement

### PR DESCRIPTION
**Pull request by alwaysintreble**
## What is this fixing or adding?
Made the tutorial a bit clearer and added file path for linux users. Removed an issue that was fixed in a newer courier release. Fixed a bad link.

## How was this tested?
Reading and clicking the links

### CODE REVIEW

File: worlds/messenger/docs/en_The Messenger.md [hunk 1]
```diff
@@ -48,25 +48,23 @@ for it. The groups you can use for The Messenger are:
...
+        is recommended to quit to title and reload the save
 * After reaching ninja village a teleport option is added to the menu to reach it quickly
 * Toggle Windmill Shuriken button is added to option menu once the item is received
 
 ## Currently known issues
 * Necro cutscene will sometimes not play correctly, but will still reward the item
 * Ruxxtin Coffin cutscene will sometimes not play correctly, but will still reward the item
-* If you receive the Fairy Bottle while in Quillshroom Marsh, The Decurse Queen cutscene will not play
+* If you receive the Fairy Bottle while in Quillshroom Marsh, The Decurse Queen cutscene will not play. You can exit
+to Searing Crags and re-enter to get it to play correctly.
```
Comment (black-sliver) [inline worlds/messenger/docs/en_The Messenger.md:63]:
```suggestion
  to Searing Crags and re-enter to get it to play correctly.
```

File: worlds/messenger/docs/setup_en.md [hunk 2]
```diff
@@ -1,31 +1,29 @@
...
-
 ## Installation
 
 1. Download and install Courier Mod Loader using the instructions on the release page
+   * [Latest release is currently 0.7.1](https://github.com/Brokemia/Courier/releases)
 2. Download and install the randomizer mod
-     * Download the latest `TheMessengerRandomizer.zip`
-     * Extract the zip file to `TheMessenger/Mods/` of your game's install location
-     * Optionally, Backup your save game
+   1. Download the latest [TheMessengerRandomizer.zip](https://github.com/minous27/TheMessengerRandomizerMod/releases)
```
Comment (black-sliver) [inline worlds/messenger/docs/setup_en.md:16]: This matches more closely what we have elsewhere. (I don't have a strong opinion). Above (Courier) has a similar situation
```suggestion
   1. Download the latest TheMessengerRandomizer.zip from the [The Messenger Randomizer Mod releases page](https://github.com/minous27/TheMessengerRandomizerMod/releases)
```

File: worlds/messenger/docs/en_The Messenger.md (hunk 1 above)
Comment (alwaysintreble) [reply worlds/messenger/docs/en_The Messenger.md:63]: wait but none of the other ones are indented, this is inconsistent.

Comment (black-sliver) [reply worlds/messenger/docs/en_The Messenger.md:63]: oh, should've rejected the original PR then. Text should be aligned in lists.

File: worlds/messenger/docs/en_The Messenger.md [hunk 3]
```diff
@@ -48,25 +48,23 @@ for it. The groups you can use for The Messenger are:
...
-    * This can cause issues if used at specific times. Current known:
-        * During Boss fights
-        * After Courage Note collection (Corrupted Future chase)
-            * This is currently an expected action in logic. If you do need to teleport during this chase sequence, it
-is recommended to quit to title and reload the save
+  * This can cause issues if used at specific times. Current known:
+    * During Boss fights
+      * After Courage Note collection (Corrupted Future chase)
+        * This is currently an expected action in logic. If you do need to teleport during this chase sequence, it
+          is recommended to quit to title and reload the save
```
Comment (black-sliver) [inline worlds/messenger/docs/en_The Messenger.md:55]:
```suggestion
    * During Boss fights
    * After Courage Note collection (Corrupted Future chase)
      * This is currently an expected action in logic. If you do need to teleport during this chase sequence, it
        is recommended to quit to title and reload the save
```

**Review by black-sliver — APPROVED** [review]

**Pull request merged**